internal/driver/glfw/menu_darwin.go

```diff
@@ -86,10 +86,13 @@ func createNativeMenu(w *window, menu *fyne.Menu, nextItemID int) (unsafe.Pointe
 
 func registerCallback(w *window, item *fyne.MenuItem, nextItemID int) int {
 	if !item.IsSeparator {
-		if action := item.Action; action != nil { // catch action value
-			callbacks = append(callbacks, func() { w.queueEvent(action) })
-			nextItemID++
-		}
+		action := item.Action // catch action value
```

Comment:
I'm unsure here, the action is being captured on menu creation so it cannot change - should instead the item be captured so the latest value for Action is used in the callback? I only noticed it as the nil check is on a value that doesn't change.